packages/core/src/extensions/SuggestionMenu/SuggestionPlugin.ts

```diff
@@ -85,7 +85,7 @@ class SuggestionMenuView<
 
     this.pluginState = stopped ? prev : next;
 
-    if (stopped || !this.editor.isEditable) {
+    if (stopped || !this.editor.isEditable || this.pluginState?.composing) {
```

Comment:
I wish that when user input between compositionStart to compositionEnd,the suggestMenu should always open.But here colsed the menu.Like the code follows: 
```
...
let _isComposing = false; 
export class SuggestionMenuProseMirrorPlugin<
...
apply(transaction, prev, _oldState, newState): SuggestionPluginState {
          if (_isComposing) {
            return prev;
          }
...
props: {
        handleDOMEvents:{
          compositionstart:(view)=>{
            console.log('run compositionstart');
            _isComposing = true;
            view.dispatch(view.state.tr.setMeta("isComposing", true));
            return false;
          },
          compositionend:(view)=>{
            console.log('run compositionend');
            _isComposing = false;
            view.dispatch(view.state.tr.setMeta("isComposing", false));
            return false;
          }
        },
        handleTextInput(view, _from, _to, text) {
...
```

Reply:
@ClytzeL : I have updated the code. Can you please test again ?